src/components/PhoneNumberInput/style.css

```diff
@@ -37,5 +41,7 @@
 }
 
 .phoneNumberContainer input {
+  @base-width: 294;
   border-bottom: none;
+  .width-percentage(@base-width, @parent-width);
```

Comment:
does this element need a width?

Reply:
Yes, I think it does, otherwise it doesn't resize